src/pages/Controllers/Controllers.js

```diff
@@ -147,9 +145,7 @@ function Details() {
         <div className="controllers--register">
           <button
             className="p-button--positive"
-            onClick={() => {
-              setShowRegisterAController(!showRegisterAController);
-            }}
+            onClick={() => history.push("/controllers/register")}
```

Comment:
I feel like we should use a querystring for the panels...That way we can open up any panel over any route.

Thoughts?

Reply:
Then we can't use React Router for this purpose.

Reply:
I don't think we need to complicate the UI with query params to forms as we have moved all details to there own page. I thought we had saved ourselves the pain of handling panels with query strings?

Reply:
Ok I see, we are using the URL as a global state store.